ssl/my-ssl.py

```diff
@@ -0,0 +1,114 @@
+import socket
```

Comment:
What does `my` in the file name stands for?

Reply:
Oh this is just a testing script! Just understood it. Let's remove this at the end then.